src/components/Login.js

```diff
@@ -0,0 +1,129 @@
+import {
+  VStack,
+  Heading,
+  Button,
+  Box,
+  FormControl,
+  InputGroup,
+  InputLeftElement,
+  InputRightElement,
+  Input,
+  FormHelperText,
+} from '@chakra-ui/react';
+import { useState } from 'react';
+import { useAuthContext } from './AuthContextProvider';
+import { Redirect } from 'react-router-dom';
+
+const LoginRegister = () => {
+  const [showPassword, setShowPassword] = useState(false);
+  const [emailValue, setEmailValue] = useState('');
+  const [passwordValue, setPasswordValue] = useState('');
+  const { handleLogin, isLoggedIn, setActiveOption, errorVisible } = useAuthContext();
+
+  const handleShowPasswordChange = () => {
+    setShowPassword(!showPassword);
+  };
+
+  return (
+    <div>
+      {!isLoggedIn ? (
+        <VStack w="100%" h="100vh" mx="left" bgGradient="linear(to-r, #E8F6EF, #E8F6EF)">
```

Comment:
Better to write it like this:

```suggestion
  const handleShowPasswordChange = () => {
    setShowPassword(!showPassword);
  };
  
  if (loggedIn) {
      return <Redirect to="/" />
  }

  return (
    <div>
        <VStack w="100%" h="100vh" mx="left" bgGradient="linear(to-r, #E8F6EF, #E8F6EF)">
```